### PR TITLE
Revert changes done for introducing a gateway proxy to infosec endpoints

### PR DESCRIPTION
**Pull request by imesh94**
## Revert changes done to introduce gateway proxy to infosec endpoints

> Proxy changes were introduced by the PR https://github.com/wso2-support/ob-compliance-toolkit-cds/pull/137. These changes are reverted because issues related to infosec data publishing from IS are resolved.

**Issue link:** *required*

**Doc Issue:** *Optional, link issue from [documentation repository](https://github.com/wso2/docs-ob/issues)*

**Applicable Labels:** *Spec, product, version, type (specify requested labels)*

------

### Development Checklist

1. [ ] Built complete solution with pull request in place.
2. [ ] Ran checkstyle plugin with pull request in place.
3. [ ] Ran Findbugs plugin with pull request in place.
4. [ ] Ran FindSecurityBugs plugin and verified report.
5. [ ] Formatted code according to WSO2 code style.
6. [ ] Have you verify the PR does't commit any keys, passwords, tokens, usernames, or other secrets?
7. [ ] Migration scripts written (if applicable).
8. [ ] Have you followed secure coding standards in [WSO2 Secure Engineering Guidelines](http://wso2.com/technical-reports/wso2-secure-engineering-guidelines)?

### Testing Checklist

1. [ ] Written unit tests.
2. [ ] Documented test scenarios(link available in guides).
3. [ ] Written automation tests (link available in guides).
4. [ ] Verified tests in multiple database environments (if applicable).
5. [ ] Verified tests in multiple deployed specifications (if applicable).
6. [ ] Tested with OBBI enabled  (if applicable).
7. [ ] Tested with specification regulatory conformance suites  (if applicable).

**Automation Test Details**

| Test Suite        | Test Script IDs   |
| ----------------- | ----------------- |
| Integration Suite | *TCXXXXX, TCXXXX* |

**Conformance Tests Details**

| Test Suite Name  | Test Suite Version | Scenarios  | Result   |
| ---------------- | ------------------ | ---------- | -------- |
| *Security Suite* | *VX.X*             | *Foo, Bar* | *Passed* |

## Resources

**Knowledge Base:** https://sites.google.com/wso2.com/open-banking/

**Guides:** https://sites.google.com/wso2.com/open-banking/developer-guides

### CODE REVIEW

**Comment by aka4rKO** [comment]
@imesh94 lets also remove the `skip_list.apis` config from the APIM toml

File: toolkits/ob-is/repository/resources/wso2is-6.0.0-deployment-cds.toml [hunk 1]
```diff
@@ -173,7 +173,6 @@ allowed_scopes = ["OB.*", "profile"]
 renew_refresh_token = false
 
 [oauth.endpoints]
-oauth2_token_url = "${carbon.protocol}://APIM_HOSTNAME:8243/token"
```
Comment (malshaniS) [inline toolkits/ob-is/repository/resources/wso2is-6.0.0-deployment-cds.toml:176]: This config should be there to resolve the audience value issue in PAR request

Comment (imesh94) [reply toolkits/ob-is/repository/resources/wso2is-6.0.0-deployment-cds.toml:176]: Fixed

**Comment by imesh94** [comment]
> @imesh94 lets also remove the `skip_list.apis` config from the APIM toml

Removed

**Review by anjuchamantha — DISMISSED** [review]

**Review by aka4rKO — APPROVED** [review]

**Comment by anjuchamantha** [comment]
@imesh94 shall we resolve the conflicts

**Review by anjuchamantha — APPROVED** [review]

**Review by aka4rKO — APPROVED** [review]

**Pull request merged**